notifier/webhook.go

```diff
@@ -74,14 +75,39 @@ func (s *Webhook) webhookURL() (string, error) {
 	return s.buildWebhookURL(url)
 }
 
-func (s *Webhook) notify(title string, message string) error {
+func (s *Webhook) webhookStatusURLs() (string, string, error) {
+	successURL := s.viper.GetString("success.url")
+	failureURL := s.viper.GetString("failure.url")
+
+	if s.buildWebhookStatusURLs == nil {
+		return successURL, failureURL, nil
+	}
+
+	return s.buildWebhookStatusURLs(successURL, failureURL)
```

Comment:
This method is no impl?

And webhookStatusURLs no error return, why need error result?